rusk-wallet/src/bin/command.rs

```diff
@@ -410,14 +419,30 @@ pub(crate) enum Command {
         amt: Dusk,
 
         /// Max amount of gas for this transaction
-        #[clap(short = 'l', long, default_value_t = DEFAULT_STAKE_GAS_LIMIT)]
+        #[clap(short = 'l', long, default_value_t = DEFAULT_LIMIT_STAKE)]
```

Comment:
as above, stake-related constant should not be used in object not related to staking

Reply:
same

Reply:
same